sr_hand_config/112/controls/host/pwm/sr_edc_effort_controllers_PWM.yaml

```diff
@@ -1,116 +1,116 @@
 # First Finger (Index finger)
 sh_lh_ffj0_effort_controller:
-  type: sr_mechanism_controllers/SlhEffortJointController
+  type: sr_mechanism_controllers/SrhEffortJointController
```

Comment:
Name is still indicating left hand `sh_[lh]_ffj0_effort_controller` but type is now right hand `S[rh]EffortJointController`. This doesn't feel right.